pike/test/copychunk.py

```diff
@@ -136,21 +169,31 @@ def generic_ssc_test_case(self, block, number_of_chunks, total_offset=0):
                 length = chunk_sz
             else:
                 length = total_len - this_offset
-            chunks.append((offset, offset+total_offset, length))
+            chunks.append((offset, offset + total_offset, length))
             this_offset += chunk_sz
 
-        fh_src, fh_dst = self._open_src_dst(src_filename, dst_filename)
+        if write_flag:
+            dst_access = access_wd
+        else:
+            dst_access = access_rwd
+
+        fh_src, fh_dst = self._open_src_dst(src_filename, dst_filename,
+                                            dst_access=dst_access)
+
+        result = self.chan.copychunk(fh_src, fh_dst, chunks, write_flag)
```

Comment:
```suggestion
        result = self.chan.copychunk(fh_src, fh_dst, chunks, write_flag=write_flag)
```

you need to pass write_flag as a keyword argument. as it is written, the boolean flag gets passed as the resume key, given the function signature for copychunk is:

```
def copychunk(self, source_file, target_file, chunks, resume_key=None, write_flag=False):
```